src/app/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="container" *ngIf="(authService.user | async)">
+
+<p>user-profile works!</p>
+
+<p>{{profile?.loginType}} login</p>
```

Comment:
Why all this optionals?

Reply:
It will give me the following error if I don't put the "?" safe navigation operator. 

Cannot read property 'loginType' of undefined

Reply:
The operator don't allow angular to throw any error if data is not present, and allow data to display asynchronously